dspace-api/src/main/java/org/dspace/xmlworkflow/storedcomponents/PoolTask.java

```diff
@@ -130,4 +138,29 @@ public void setActionID(String actionID) {
     public String getActionID() {
         return this.actionId;
     }
+
+    @Override
+    public String getTypeText() {
+        return "pooltask";
```

Comment:
Should this be in `Constants`?

Reply:
done https://github.com/DSpace/DSpace/pull/2312/files#diff-52c496a60902e078268287bedbe773ffR144